api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMaker2MirrorSpec.java

```
@@ -0,0 +1,120 @@
+@EqualsAndHashCode
+public class KafkaMirrorMaker2MirrorSpec implements Serializable, UnknownPropertyPreserving {
+    private static final long serialVersionUID = 1L;
+
+    private String sourceCluster;
+    private String targetCluster;
+    private KafkaConnectorSpec sourceConnector;
+    private KafkaConnectorSpec checkpointConnector;
+    private KafkaConnectorSpec heartbeatConnector;
+    private String topics;
+    private String groups;
+    private Map<String, Object> additionalProperties;
+
+    @Description("A regular expression matching the topics to be mirrored.")
+    @JsonProperty(required = true)
+    public String getTopics() {
+        return topics;
+    }
+
+    public void setTopics(String topics) {
+        this.topics = topics;
+    }
+
+    @Description("A regular expression matching the consumer groups to be mirrored.")
+    public String getGroups() {
+        return groups;
+    }
+
+    public void setGroups(String groups) {
+        this.groups = groups;
+    }
+
+    @Description("The alias of the source cluster used by the Kafka MirrorMaker 2.0 connectors. The alias must match a cluster in the list at spec.clusters.")
+    @JsonProperty(required = true)
+    public String getSourceCluster() {
+        return sourceCluster;
+    }
+
+    public void setSourceCluster(String sourceCluster) {
+        this.sourceCluster = sourceCluster;
+    }
+
+    @Description("The alias of the target cluster used by the Kafka MirrorMaker 2.0 connectors. The alias must match a cluster in the list at spec.clusters.")
+    @JsonProperty(required = true)
+    public String getTargetCluster() {
+        return targetCluster;
+    }
+
+    public void setTargetCluster(String targetCluster) {
+        this.targetCluster = targetCluster;
+    }
+
+    @Description("The specification of the Kafka MirrorMaker 2.0 source connector. The connector class name is automatically set to `org.apache.kafka.connect.mirror.MirrorSourceConnector`.")
```

What does ``The connector class name is automatically set to `org.apache.kafka.connect.mirror.MirrorSourceConnector`.`` mean? That that's the _default_? If so, then just say "Defaults to ...".

---

`KafkaConnectorSpec` has a className property that the MM2 operator sets to `org.apache.kafka.connect.mirror.MirrorSourceConnector` and throws an exception if you try to set it to something else (see https://github.com/strimzi/strimzi-kafka-operator/pull/2311/files#diff-255b3c5e397f185aefe6a3e223239e59R247-R252), so I think calling it a default would mislead users into thinking they could override it, which they can't.

I have changed it to:

    "The specification of the Kafka MirrorMaker 2.0 source connector. 
    The connector class name is set to 
    `org.apache.kafka.connect.mirror.MirrorSourceConnector`."